src/main/java/seedu/exchangecoursemapper/command/AddCoursesCommand.java

```diff
@@ -30,29 +30,34 @@ public String trimString(String string) {
         String trimmedString = string.trim();
         String[] outputSubstrings = trimmedString.split(" ", 2);
 
+        //throws an error when there is no user input after the add command.
         if (outputSubstrings.length < 2 || outputSubstrings[1].trim().isEmpty()) {
-            throw new IllegalArgumentException("Please provide a valid course name!");
+            throw new IllegalArgumentException("Please provide the nus course code, " +
+                    "name of partner university (PU) and the PU course code.");
```

Comment:
I think this code works well! But perhaps we can also use the Exception class we have in our code to print the exception message? For example:
```
System.err.println(Exception.MissingCourseInformation());
```